Makefile

```diff
@@ -57,6 +61,21 @@ lint: fmt vet generate ## All-in-one linting
 generate: ## Generate additional code and artifacts
 	@go generate ./...
 
+.PHONY: install-crd
+install-crd: export KUBECONFIG = $(KIND_KUBECONFIG)
+install-crd: generate ## Install CRDs into cluster
+	kubectl apply -f package/crds
+
+.PHONY: install-samples
+install-samples: export KUBECONFIG = $(KIND_KUBECONFIG)
+install-samples: generate install-crd ## Install samples into cluster
+	kubectl apply -f package/samples
```

Comment:
This fails because `package/samples` doesn't exist when running `make install-samples`. But a folder `samples` exist.